docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -0,0 +1,369 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# sofar and SOFA\n",
+    "\n",
+    "If you are new to SOFA and/or sofar, this is a good place to start. SOFA is short for *Spatially Oriented Format for Acoustics* and is an open file format for saving acoustic data, for example head-related impulse responses (HRIRs). Good places to get more information about SOFA after this introduction are\n",
+    "\n",
+    "* The [SOFA paper](https://doi.org/10.17743/jaes.2022.0026)\n",
+    "* The documentation of the [SOFA conventions](https://sofar.readthedocs.io/en/stable/resources/conventions.html)\n",
+    "* [sofaconventions.org](https://www.sofaconventions.org)\n",
+    "* The SOFA standard [AES69-2022](https://www.aes.org/publications/standards/search.cfm?docID=99)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating SOFA objects\n",
+    "\n",
+    "To cover a variety of data, SOFA offers different *conventions*. A convention defines, what data can be saved and how it is saved. You should always find the most specific convention for your data. This will help you to identify relevant data and meta data that you should provide along the actual acoustic data. Using sofar, a list of possible conventions can be obtained with"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sofar as sf\n",
+    "import os\n",
+    "\n",
+    "sf.list_conventions()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us assume, that you want to store head-related impulse responses (HRIRs). In this case the most specific convention is *SimpleFreeFieldHRIR*. To create a SOFA object use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa = sf.Sofa('SimpleFreeFieldHRIR')\n",
+    "print(sofa)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The return value `sofa` is a `sofar.Sofa` object filled with the default values of the `SimpleFreeFieldHRIR` convention. Note that you can also get a sofa object that has only the mandatory attributes by calling `sf.Sofa('SimpleFreeFieldHRIR', mandatory=True)`. However, it is recommended to start with all attributes and discard empty optional attributes before saving the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting information about SOFA objects\n",
+    "\n",
+    "An formal description of each convention is given in the [sofar documentation](https://sofar.readthedocs.io/en/stable/resources/conventions.html). You might have noted from the documentation that three different kinds of data types can be stored in SOFA files:\n",
+    "\n",
+    "* **Attributes:**\n",
```

Comment:
maybe a codesnipped liek ``sofa.inspect``is helpful to see what is inside the object first.

Reply:
Good idea, I changed the order of the paragraph to explain the concepts more closely related to the example.